README.md

```diff
@@ -1 +1,2054 @@
+
+// converts a Mailchimp date to a Date object
+var date = pkg.mailchimp.dates.toDate('2017-07-19T13:12:56+00:00');
+
+// converts milliseconds to a Mailchimp date
+var mailchimpDate = pkg.mailchimp.dates.fromMillis(1500469976000);
+
+// converts a Date object to a Mailchimp date
+var mailchimpDate = pkg.mailchimp.dates.fromDate(new Date());
+```
+
+### Email helper
+
+```
+var md5 = pkg.mailchimp.emailHash(email);
+```
+
+Returns the MD5 hash of the email. This is needed when you need to work with members in list.
+
+
+## Events
+
+### Webhook
+
+Changes in lists like member subscriptions or unsubscriptions will be notified through this event. Keep in mind that you
+need to configure webhooks on each list. See the documentation [here](http://kb.mailchimp.com/integrations/api-integrations/set-up-webhooks)
+for more information.
+
+According to the documentation of the [format of webhooks](http://developer.mailchimp.com/documentation/mailchimp/guides/about-webhooks/)
+you should get key-value pairs like when there is an HTML form submission. However, to make it easier, we convert it to
+JSON. For example, instead of getting this:
+
+```
+"type": "subscribe", 
+"fired_at": "2009-03-26 21:35:57", 
+"data[id]": "8a25ff1d98", 
+"data[list_id]": "a6b5da1054",
+"data[email]": "api@mailchimp.com", 
+"data[email_type]": "html", 
+"data[merges][EMAIL]": "api@mailchimp.com", 
+"data[merges][FNAME]": "MailChimp", 
+"data[merges][LNAME]": "API", 
+"data[merges][INTERESTS]": "Group1,Group2", 
+"data[ip_opt]": "10.20.10.30", 
+"data[ip_signup]": "10.20.10.30"
+```
+
+You will get this:
+
+```js
+{
+    type: "subscribe", 
+    fired_at: "2009-03-26 21:35:57", 
+    data: {
+        id: "8a25ff1d98", 
+        list_id: "a6b5da1054",
+        email: "api@mailchimp.com", 
+        email_type: "html", 
+        merges: {
+            EMAIL: "api@mailchimp.com", 
+            FNAME: "MailChimp", 
+            LNAME: "API", 
+            INTERESTS: "Group1,Group2"
+        },
+        ip_opt: "10.20.10.30", 
+        ip_signup: "10.20.10.30"
+    }
+}
+```
+
+## About SLINGR
+
+SLINGR is a low-code rapid application development platform that accelerates development, with robust architecture for integrations and executing custom workflows and automation.
+
+[More info about SLINGR](https://slingr.io)
+
+## License
+
+This package is licensed under the Apache License 2.0. See the `LICENSE` file for more details.
+
+# MailChimp v1.0.0
+
+# Javascript API
+
+The Javascript API of the mailchimp package has three pieces:
+
+- **HTTP requests**: These allow to make regular HTTP requests.
+- **Shortcuts**: These are helpers to make HTTP request to the API in a more convenient way.
+- **Additional Helpers**: These helpers provide additional features that facilitate or improves the package usage in SLINGR.
+
+## HTTP requests
+You can make `PATCH`,`POST`,`GET`,`DELETE`,`PUT` requests to the [mailchimp API](API_URL_HERE) like this:
```

Comment:
Please change the `API_URL_HERE` variable to `https://mailchimp.com/developer/marketing/docs/fundamentals/` , it is a bug that has been left in some endpoint standardizations.